frontend/src/api/index.ts

```diff
@@ -39,28 +41,42 @@ type _Geometry = {
   wkb_geometry: string;
 }[];
 
-// get geojson from geography data
-export async function getGeometry(type: string, idField: string) {
-  const data = await request<_Geometry>(`${api}/${type}`);
-
-  // transform data into desired format
-  return data.map(
-    ({ wkb_geometry, ...d }) =>
-      ({
-        type: "Feature",
-        geometry: JSON.parse(wkb_geometry) as GeoJsonGeometry,
-        properties: {
-          ...d,
-          id: d[idField],
-          name: d.full || d.name || "",
-        },
-      }) satisfies Feature,
-  );
+/** data geojson properties fields */
+type DataProps = {
+  id: string | number | undefined;
+  name: string;
+  full?: string | undefined;
+  fips?: string | undefined;
+  us_fips?: string | undefined;
+  objectid: number;
+  ogc_fid: number;
+};
+
+/** get geojson from geography data */
+export async function getData(
+  type: string,
+  idField: string,
+): Promise<FeatureCollection<Geometry, DataProps>> {
+  const data = await request<_Data>(`${api}/${type}`);
+
+  /** transform data into desired format */
+  return {
```

Comment:
Reformatting this to return feature collection instead of array of features, to make it pure geojson.